sdk/computeschedule/Azure.ResourceManager.ComputeSchedule/samples/Generated/Samples/Sample_SubscriptionResourceExtensions.cs

```diff
@@ -22,7 +22,7 @@ public partial class Sample_SubscriptionResourceExtensions
         public async Task SubmitVirtualMachineDeallocate_ScheduledActionsVirtualMachinesSubmitDeallocate()
         {
             // Generated from example definition: 2024-10-01/ScheduledActions_VirtualMachinesSubmitDeallocate.json
-            // this example is just showing the usage of "ScheduledActions_SubmitVirtualMachineDeallocate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "ScheduledActions_virtualMachinesSubmitDeallocate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
It's due to **Not** honor `@clientName`:
https://github.com/Azure/azure-rest-api-specs/blob/95aa73b8831deb2940ac7067b8f296cd189a5fe1/specification/computeschedule/ComputeSchedule.Management/client.tsp#L9-L12